src/components/AvatarWithImagePicker.js

```diff
@@ -23,6 +23,8 @@ import getImageResolution from '../libs/fileDownload/getImageResolution';
 import PressableWithoutFeedback from './Pressable/PressableWithoutFeedback';
 import DotIndicatorMessage from './DotIndicatorMessage';
 import * as Browser from '../libs/Browser';
+import withNavigationFocus from './withNavigationFocus';
```

Comment:
NAB because I see we're not consistent with this, but I think we should spread withNavigationFocusPropTypes into the component prop types.